getting-started/git-boot-camp.rst

```diff
@@ -669,3 +669,65 @@ Examples of useful commands:
 * Set the browser::
 
       $ gh config set browser <browser-path>
+
+
+Git worktree
+------------
+
+When working on several version branches at once, experienced Git users can
+take advantage of Git's worktree to check out several branches at once.
+
+Setting up Git worktree
+^^^^^^^^^^^^^^^^^^^^^^^
+
+With an existing CPython clone (see :ref:`clone-your-fork`), rename the
+``cpython`` directory to ``main`` and move it into a new ``cpython``
+directory, so we have a structure like:
+
+.. Generated with: tree -L 1 -d cpython
+
+.. code-block:: text
+
+   cpython
+   └── main (.git is here)
+
+Next, create worktrees for the other branches::
```

Comment:
perhaps add a few words about the relationship between the different worktrees? (e.g. do they share remotes? git config? if you fetch in one is it visible in others? if you add/rename remotes in one is it visible in others?)

Reply:
In general this stuff is shared. Yes, remotes are shared, and (by default) repo config. You can add/rename a remote from one and cd to another, and still see them all.

https://git-scm.com/docs/git-worktree details some things which are not shared, which is likely way too detailed to include here. There's a link to the manual in the "See also" at the end.

Reply:
Indeed the first paragraph could do a better job at clarifying *why* I would want to use worktrees.  The ability to work on multiple branches in parallel without switching back and forth from the same clone/dir is one.  AFAIU, other advantages should be:
* there's only one copy of the repo (`.git/*`)
* you only need to set up one repo (e.g. create remotes)
* you only need to update one repo (i.e. pull changes)
* less disk space is needed

Reply:
This sounds good, would you like to suggest a paragraph?

Reply:
Done!